tests/core/test_common_utils.py

```diff
@@ -42,7 +42,7 @@ def test_create_one_dim_tr_model():
         },
         "algorithm": {
             "learned_rewards": True,
-            "terget_is_delta": True,
+            "target_is_delta": True,
```

Comment:
Good catch! Was the test failing for you? This test runs fine for me even with the typo. Can you share the error you were getting? I wonder if the difference is due to a Hydra version (my version just sets `cfg.algorith.target_is_delta = None`.

Reply:
Oh sorry, I missed that you had posted the error in the description. Probably some changes in a new version of OmegaConf, thanks for the fix!

Reply:
No problem, thanks for the library!